test/functional/cmdline_options_testresources/src_90/com/ibm/j9/getcallerclass/ReflectionMHTests.java

```diff
@@ -62,7 +62,8 @@ public static boolean test_getCallerClass_Helper_Reflection_fromBootExtWithAnnot
 			cls = (Class<?>) method.invoke(null, new Object[0]);
 
 			boolean isClassNameExpected;
-			if (VersionCheck.major() >= 18) {
+			String useDMHStr = System.getProperties().getProperty("jdk.reflect.useDirectMethodHandle");
+			if (VersionCheck.major() >= 18 && (useDMHStr != null && useDMHStr.equals("true"))) {
```

Comment:
```suggestion
			if (VersionCheck.major() >= 18 && Boolean.getBoolean("jdk.reflect.useDirectMethodHandle")) {
```
L65 can be removed.

Reply:
Thanks a lot, I have applied the recommendation in https://github.com/eclipse-openj9/openj9/compare/7f66c8ff8a083767606602b2700d5fb56ad462d9..b58d7f658a0a54755bde06d23247ae4e544796f5